src/node_input_factory/node_input_classes.py

```diff
@@ -57,11 +57,17 @@ def __init__(self, location: str, name: str, node_purpose: str):
 
 class EngineNodeInput(NodeInput):
     # Enum 3
-    def __init__(self, set_engine: bool, name: str, node_purpose: str):
+    def __init__(self, set_engine: int, name: str, node_purpose: str):
         super().__init__(set_engine, name, node_purpose)
 
 
 class TemperatureNodeInput(NodeInput):
     # Enum 4
     def __init__(self, temperature: float, name: str, node_purpose: str):
         super().__init__(temperature, name, node_purpose)
+
+
+class ServoNodeInput(NodeInput):
```

Comment:
Waarom heb je een servo class gemaakt? Hoort dit niet bij de "steering node"

Reply:
> Ik denk dat er een verwarring is ontstaan:
> 
> Op een vehicle hebben we een "Steering" node, wat een invoer en een uitvoer heeft.
> 
> Je invoer wordt gelezen van een "encoder". Dit heeft als invoer de werkelijke hoek van de stuurophanging. Dit kan bijvoorbeeld een potmeter zijn, zoals wat ik gebruik bij de hardware test, maar er zijn andere mogelijkheden.
> 
> Je uitvoer wordt uitgevoerd door een actuator. Dit is dus wat fysiek de stuurophanging beweegt zodat de voertuig links, rechts, rechtdoor rijdt. In geval van de smarterdam zijn dit twee servo motoren. In geval van mijn kleine hardware demo, is dit een enkele, losse servo.
> 
> Is dit misschien de reden waarom je een losse servo node heb gemaakt?

Ja, ik heb dit toegevoegd vanwege de demo. Ik zal wel commentaar erbij toevoegen.

Reply:
Kunnen we het niet beter andersom doen, dat de demo een steering node wordt ipv dat er een nieuwe node soort gedefineerd wordt?

Reply:
ja is goed ik zal het aanpassen inprincipe doen ze hetzelfde